test/tpu/pjrt_test_job.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
```

Comment:
i wonder what this yaml file does

Reply:
This defines the kubernetes workload that the cloud build executes

Reply:
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/